pyshelf/bulk_update/utils.py

```diff
@@ -13,6 +14,40 @@ def run(args):
                 options provided to the update-search-index
                 script
     """
+    container = _configure(args, "update-search-index")
+    bucket_string = args.get("--bucket")
```

Comment:
I don't think this one is necessary.  Is there a situation where this would make a difference?  Does it change any functionality to pass this along for the `Cleaner` stuff?